package.json

```diff
@@ -27,6 +28,11 @@
     "babel-preset-stage-0": "^6.5.0",
     "browser-sync": "^2.12.10",
     "browser-sync-webpack-plugin": "^1.0.1",
+    "eslint": "^3.0.1",
```

Comment:
Confirmed that I still get errors on `npm install`, however the linter seems to run. Should I be okay with this?